### DISCUSSION TITLE
How can we access the location of the device using IMEI and phone number?

### DISCUSSION THREAD

**Discussion by handesaglam** [Ludy87/xplora_watch#435] [General]
How can we access the location of the device using IMEI and phone number? Is it provided with API etc.? Can you please help? I use vo922s

**Comment by Ludy87** [comment]
Hi @handesaglam , I don't understand the question correctly.The integration must be installed/added as described. You must then verify yourself with a telephone number or email address and password. After that, location tracking should be possible. Do you own an xplora watch?

**Reply by handesaglam** [reply]
What I mean is, how is the communication between the watch and the phone achieved?

**Reply by Ludy87** [reply]
Communication takes place via the Xplora interface. Just like in the Xplora app.

**Comment by handesaglam** [comment]
Let me explain briefly like this: I have a v0922 s model smart watch. I
need to show the location of the watch on the phone. How can I do this in
an app? Is there a web service or something that shows the location of the
watch?

Ludy ***@***.***>, 26 Haz 2024 Çar, 12:13 tarihinde şunu
yazdı:

**Reply by Ludy87** [reply]
If it's not an Xplora watch, I can't help you.

**Comment by handesaglam** [comment]
What I mean is, which service does xplora use to show the position of the
watch on the back of the phone?

Hande Saglam ***@***.***>, 26 Haz 2024 Çar, 17:42 tarihinde
şunu yazdı:

**Reply by Ludy87** [reply]
Since I'm not a developer at Xplora, I don't know.

**Comment by handesaglam** [comment]
Ok, no problem.

Ludy ***@***.***>, 26 Haz 2024 Çar, 18:17 tarihinde şunu
yazdı:
